### ISSUE TITLE
I can't Set 3-hour interval with x-axis

### ISSUE THREAD

**Issue by zengyuehao** [apache/echarts#15266]
### Version
5.1.2

### Steps to reproduce
 set my X-axis type to time, but I can't Set 3-hour interval with x-axis. I've tried to add anchors ro series and set max-interval, but it can only be changed to one hour, two hours and four hour intervals

### What is expected?
x-axis ：The time interval is 3 hours

### What is actually happening?
only change to one hour, two hours and four hour intervals

**Comment by looserLiu** [comment]
i also have the problem

**Comment by ob-ARC** [comment]
Same problem here.

**Comment by wiwth** [comment]
>I'm sorry to bother you, I want to ask if you have solved this problem now？

**Comment by ob-ARC** [comment]
> > I'm sorry to bother you, I want to ask if you have solved this problem now？

Nope, I guess we need to wait for the implementation of the aforementioned feature #17576

**Comment by Ting-alphacore** [comment]
can't set 3 hours too, is the problem solved?
